autogen/oai/openai_utils.py

```diff
@@ -368,6 +373,7 @@ def filter_config(config_list, filter_dict):
             {'model': 'gpt-3.5-turbo'},
             {'model': 'gpt-4'},
             {'model': 'gpt-3.5-turbo', 'api_type': 'azure'},
+            {'model': 'gpt-3.5-turbo', 'tags': ['gpt35_turbo', 'gpt-35-turbo']},
```

Comment:
Maybe change the example to use `'tags': ['deployment_1', 'deployment_2']` or something similar instead? Having model name and tags so similar might be a little bit confusing at a quick glance.

Reply:
As of now Azure's deployment name is effectively the "model" name used in configs.